receiver/elasticsearchreceiver/testdata/integration/Dockerfile.elasticsearch.7_16_3

```diff
@@ -4,4 +4,4 @@ ENV discovery.type=single-node
 
 ENV ES_JAVA_OPTS='-Xms512m -Xmx512m'
 
-EXPOSE 9300
+EXPOSE 9200
```

Comment:
Is this change expected?

Reply:
Yeah, this is part of standardizing the way we manage ports. Previously, this version was assigned a unique port here, while the other versions used elasticsearch's standard 9200. The new pattern delegates the responsibility to the `testcontainers` package.